Python/Tests/DebuggerTests/BaseDebuggerTests.cs

```diff
@@ -29,6 +29,7 @@
 using TestUtilities;
 
 namespace DebuggerTests {
+    [TestClass, Ignore]
```

Comment:
Is this necessary? This isn't actually a test class.

Reply:
I did it to be consistent with "DebuggerTests" class. Upon further investigation, I see that it is not required and I've removed it. Will be updated in next commit.